.gitignore

```diff
@@ -1,3 +1,4 @@
 /bin
 .env
 .kube/
+*.coverprofile
```

Comment:
These get generated in every directory when you run ginkgo / go test with `-cover`